app/src/main/java/ar/com/wolox/android/training/ui/login/LogInPresenter.java

```diff
@@ -20,13 +20,14 @@
     private RetrofitServices mRetrofitServices;
 
     // Constructor
-    public LogInPresenter(LogInView viewInstance) {
+    public LogInPresenter(LogInView viewInstance, RetrofitServices retrofitServices) {
```

Comment:
Instead of adding a comment you can create a `javadoc` like this:
```java
/**
* Constructor. Information about what it does or how does it work
*/
public LogInPresenter(....) {
```